### PR TITLE
sh tools/substitutions.sh

### PR DESCRIPTION
**Pull request by themr0c**
Asciidoc attributes substitutions.

### CODE REVIEW

File: src/main/pages/che-7/administration-guide/assembly_collecting-logs-using-chectl.adoc [hunk 1]
```diff
@@ -14,10 +14,10 @@ ifdef::context[:parent-context-of-collecting-logs-using-chectl: {context}]
 = Collecting logs using chectl
 
 
-It is possible to get all Eclipse Che logs from k8s cluster using `chectl` tool
+It is possible to get all {prod} logs from k8s cluster using `chectl` tool
```
Comment (rkratky) [inline src/main/pages/che-7/administration-guide/assembly_collecting-logs-using-chectl.adoc]:
```suggestion
It is possible to get all {prod} logs from a Kubernetes cluster using the `chectl` tool.
```

File: src/main/pages/che-7/administration-guide/assembly_collecting-logs-using-chectl.adoc [hunk 2]
```diff
@@ -14,10 +14,10 @@ ifdef::context[:parent-context-of-collecting-logs-using-chectl: {context}]
...
 
-- `chectl server:start` automatically starts collecting Eclipse Che servers logs during installation of Eclipse Che
-- `chectl server:logs` collects existed Eclipse Che servers logs
+- `chectl server:start` automatically starts collecting {prod} servers logs during installation of {prod}
+- `chectl server:logs` collects existed {prod} servers logs
```
Comment (rkratky) [inline src/main/pages/che-7/administration-guide/assembly_collecting-logs-using-chectl.adoc]:
```suggestion
- `chectl server:logs` collects existing {prod} server logs
```

File: src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc [hunk 3]
```diff
@@ -91,20 +91,20 @@ The link:https://github.com/eclipse/che-plugin-registry/tree/master/v3/plugins[c
...
 
-.Example `meta.yaml` for a Che-Theia plug-in: the Che machine-exec Service
+.Example `meta.yaml` for a Che-Theia plug-in: the {prod-short} machine-exec Service
 [source,yaml]
 ----
 apiVersion: v2
 category: Other
-description: Che Plug-in with che-machine-exec service to provide creation terminal or tasks for Eclipse CHE workspace containers.
-displayName: Che machine-exec Service
+description: {prod-short} Plug-in with che-machine-exec service to provide creation terminal or tasks for Eclipse CHE workspace containers.
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc]:
```suggestion
description: {prod-short} plug-in with che-machine-exec service to provide creation terminal or tasks for {prod} workspace containers.
```

File: src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc [hunk 4]
```diff
@@ -91,20 +91,20 @@ The link:https://github.com/eclipse/che-plugin-registry/tree/master/v3/plugins[c
...
+displayName: {prod-short} machine-exec Service
 firstPublicationDate: "2019-12-02"
 icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
 name: che-machine-exec-plugin
 publisher: eclipse
 repository: https://github.com/eclipse/che-machine-exec/
 title: Che machine-exec Service Plugin
-type: Che Plugin
+type: {prod-short} Plugin
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc]:
```suggestion
type: {prod-short} plug-in
```

File: src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc [hunk 5]
```diff
@@ -20,15 +20,15 @@ The link:https://github.com/eclipse/che-plugin-registry/tree/master/v3/plugins[c
...
+`type`: `{prod-short} Plugin`, `VS Code extension`.
 `version`: Version (example\: `7.5.1`).
 `spec`: Specifications (see underneath).
 :===
 
 .`spec` attributes
 :===
 `endpoints`: plugin endpoints
-`containers`: sidecar containers for plugin. `Che Plugin` and `VS Code extension` supports only one container.
+`containers`: sidecar containers for plugin. `{prod-short} Plugin` and `VS Code extension` supports only one container.
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc]:
```suggestion
`containers`: sidecar containers for plug-in. `{prod-short} plug-in` and `VS Code extension` supports only one container.
```

File: src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc (hunk 4 above)
Comment (rkratky) [reply src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc]: Hmmm... looking at this again... is `Che Plugin` a defined 'type'? If it is, then neither an attribute for prod name, nor the 'plug-in' form should be used.

**Review by rkratky — APPROVED** [review]

File: src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc (hunk 4 above)
Comment (themr0c) [reply src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc]:
```suggestion
type: Che Plugin
```

Comment (themr0c) [reply src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc]: `type: Che Plugin` indeed see this example: https://github.com/eclipse/che-plugin-registry/blob/master/v3/plugins/eclipse/che-machine-exec-plugin/7.7.0/meta.yaml

File: src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc (hunk 5 above)
Comment (themr0c) [reply src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc]: I have to revert `Che Plugin` and `Che editor` types.

Comment (themr0c) [reply src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc]: This is confusing: type for a `{prod-short} plug-in` can be: `Che Plugin`, `Che Editor`, `VS Code extension`. Depending on the context we may or may not use the attribute. All my attempts to automate the substitution with regex failed so far. I need to reassess everything manually each time :/.

File: src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc [hunk 6]
```diff
@@ -91,13 +91,13 @@ The link:https://github.com/eclipse/che-plugin-registry/tree/master/v3/plugins[c
...
 
-.Example `meta.yaml` for a Che-Theia plug-in: the Che machine-exec Service
+.Example `meta.yaml` for a Che-Theia plug-in: the {prod-short} machine-exec Service
 [source,yaml]
 ----
 apiVersion: v2
 category: Other
-description: Che Plug-in with che-machine-exec service to provide creation terminal or tasks for Eclipse CHE workspace containers.
-displayName: Che machine-exec Service
+description: Che Plugin with che-machine-exec service to provide creation terminal or tasks for {prod} workspace containers.
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc:99]: This sentence is giving me a headache :(

File: src/main/pages/che-7/end-user-guide/ref_devfile-reference.adoc [hunk 7]
```diff
@@ -439,7 +439,7 @@ The following environment variables are pre-set by the {prod-short} server:
 
 * `CHE_MACHINE_AUTH_SIGNATURE__ALGORITHM`: The encryption algorithm used in the secured communication with the {prod-short} server.
 
-A devfiles may only need the `CHE_PROJECTS_ROOT` environment variable to locate the cloned projects in the component's container. More advanced devfiles might use the `CHE_WORKSPACE_LOGS_ROOT__DIR` environment variable to read the logs (for example as part of a devfile command). The environment variables used to securely access the {prod-short} server are mostly out of scope for devfiles and are present only for advanced use cases that are usually handled by the {prod-short} plug-ins.
+A devfiles may only need the `CHE_PROJECTS_ROOT` environment variable to locate the cloned projects in the component's container. More advanced devfiles might use the `CHE_WORKSPACE_LOGS_ROOT__DIR` environment variable to read the logs (for example as part of a devfile command). The environment variables used to securely access the {prod-short} server are mostly out of scope for devfiles and are present only for advanced use cases that are usually handled by the {prod-short} Plugins.
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/ref_devfile-reference.adoc]: In this case, I believe it should be `{prod-short} plug-ins`.

File: src/main/pages/che-7/overview/con_che-plug-ins.adoc [hunk 8]
```diff
@@ -1,7 +1,7 @@
 [id="{prod-id-short}-plug-ins_{context}"]
-= {prod-short} plug-ins
+= `Che Plugin` plug-ins
```
Comment (rkratky) [inline src/main/pages/che-7/overview/con_che-plug-ins.adoc:2]: Again, I think this whole concept module doesn't talk about the `Che Plugin` plug-in type but about `{prod-short} plug-ins` in general.

**Review by rkratky — APPROVED** [review]

File: src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc (hunk 6 above)
Comment (themr0c) [reply src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc:99]: It is a verbatim copy of a real metadata, I am sorry :/. Should we review them?

File: src/main/pages/che-7/end-user-guide/ref_devfile-reference.adoc (hunk 7 above)
Comment (themr0c) [reply src/main/pages/che-7/end-user-guide/ref_devfile-reference.adoc]: Nice pick, thanks.

**Pull request merged**